### ISSUE TITLE
Can't log in to Audible.com

### ISSUE THREAD

**Issue by rbduck** [openaudible/openaudible#797]
In Controls>Connect to Audible, I am unable to connect to Audible.com. I get an error for my Username/password. both are correct for my Audible account. The challenge code is correct. Any Ideas? I am using OpenAudible version 3.3in Windows 10. I can log onto Audible.com from a chrome browser with no problem Thanks

**Comment by openaudible** [comment]
Do you have a user name or email for logging in? If you're using an old username, you must log in with the legacy checkmark option. If you have a regular amazon account, that box should be unchecked.

**Comment by openaudible** [comment]
Please quit and restart OA. Do a Connect to audible. Send bug report from About menu. Will dig into it.

**Comment by rbduck** [comment]
OK. I tried to open the app today. I received and error about a .jar file that is missing. I downloaded 2.3.5 today and installed that. I opened the app and was successful logging in Audible.com. I could have sworn I had already tried reinstallation. I realize there is a much later version of Openaudible. I see that there a a fee for version 3.3. I'm not cheap I'm just broke. I semiretired then caught Covid. I can't work at this time. Maybe I can get it at a later date. I'm sorry to have bothered you. Thank You!! ;))
